demosplan/DemosPlanCoreBundle/ResourceTypes/GisLayerResourceType.php

```diff
@@ -88,6 +94,15 @@ public function isAvailable(): bool
 
     public function isDeleteAllowed(): bool
     {
+        if ($this->currentUser->hasPermission('area_map_participation_area')) {
```

Comment:
how do we ensure, that there is one base layer in the beginning? A blueprint doesn't have to have a layer at all.

Reply:
While creating a procedure the layers will be copied from the master blueprint which had already a basemap layer. So one layer will be direct available